frontend/src/components/card/card.styles.js

```diff
@@ -0,0 +1,141 @@
+import styled from 'styled-components';
+
+function roundCorners() {
+    let rules = [];
+    for (let i = 0; i < 4; i++) {
+        const pageMinWidth = 363;
+        const breakpointWidth = 385;
+        const mediaQuery = `@media only screen and
+                    (min-width: ${pageMinWidth + breakpointWidth * i}px) and
+                    (max-width: ${pageMinWidth + breakpointWidth * (i + 1)}px)`;
+        const everyNth = `${i + 1}n + 1`;
+
+        rules.push(`${mediaQuery} {
```

Comment:
This looks a little bit tricky, but don't have any better alternative to suggest this point. This is just a note, no need to refactor at this point.

Reply:
Indeed. As far as I can tell, this was the only way to achieve rounded corners with CSS. Otherwise it would require JS which brings more trickiness with window resizing etc. 😕